src/components/DateRangePicker.jsx

```diff
@@ -198,6 +199,14 @@ export default class DateRangePicker extends React.Component {
     }
   }
 
+  getDateString(date) {
+    const { displayFormat } = this.props;
+    if (displayFormat) {
+      return date && date.format(displayFormat);
```

Comment:
`toLocalizedString` returns `null` for falsey dates. I'm going to go back and add some tests for that I think.